.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: node_js
+env:
+  global:
+    - firebase_bucket=gs://tbatv-prod-hrd.appspot.com
+
+branches:
+  only:
+  - master
+
+before_install:
+  - openssl aes-256-cbc -K ${ENCRYPTED_KEY_NAME} -iv ${ENCRYPTED_IV_NAME} -in gcp-service-key.json.enc -out gcp-service-key.json -d
+  - nvm install 9
+
+install:
+  - travis_retry npm install
+  - travis_retry npm install -g react-native-cli
+
+script:
+  - react-native bundle --platform ios --dev false --assets-dest ios --entry-file index.ios.js --bundle-output ios/main.jsbundle --reset-cache
+  - zip -r ios.zip ios/
+
+after_success:
+  - gcloud auth activate-service-account --key-file gcp-service-key.json
+  - gsutil cp ios.zip $firebase_bucket/react-native/ios.zip
```

Comment:
Where is this `$firebase_bucket` coming from? (Can we capitalize it, or is it set from the gcloud thing?)